cat-theme/src/css/base/_base.scss

```diff
@@ -0,0 +1,114 @@
+*,
+*::after,
+*::before {
+    box-sizing: inherit;
+    font-family: "Montserrat", sans-serif;
+    margin: 0;
+    padding: 0;
+    -webkit-box-sizing: antialiased;
+}
+
+html {
+    font-size: 62.5%; 
+    overflow-x: hidden;
+}
+
+body {
+    box-sizing: border-box;
+    padding: 0;
+    overflow-x: hidden;
+    -moz-osx-font-smoothing: grayscale;
+    -webkit-font-smoothing: antialiased;
+
+    @include respond(tab-port) {
+        padding: 0;
+    }
+}
+
+#wrapper {
+    padding: 0;
+}
+
+.main {
+    transition: 0.5s;
+}
+
+*:focus {
+    outline: none !important;
+}
+
+h2 {
+    font-family: 'Montserrat', sans-serif;
+    font-size: 4.2rem;
+    font-weight: 800;
+    letter-spacing: 0.01em;    
+
+    @include respond(phone) {
+        font-size: 2.8rem;
+    }
+}
+
+h3 {
+    letter-spacing: 0.25rem;
+    font-size: 2.2rem;
+    font-weight: 600;
+    margin-bottom: 2rem;
+
+    @include respond(phone) {
+        font-size: 1.8rem;
+    }
+}
+
+h4 {
+    letter-spacing: 0.2rem;
+    font-size: 2.5rem;
+    font-weight: 600;
+    margin-bottom: 1rem;
+}
+
+h5 {
+    font-size: 1.3rem;
+    font-weight: 600;
+    letter-spacing: 0.2rem;
+    margin-bottom: 1rem;
+    text-transform: uppercase;
+}
```

Comment:
Since there are some similarities between the `h4` and `h5` CSS rules, we should try to combine the rules together if possible and keep what is only different in a new CSS rule.

Ex.
```
h4, h5 {
    font-weight: 600;
    letter-spacing: 0.2rem;
    margin-bottom: 1rem;
}
```